dts/bindings/sensor/st,lps22hh-common.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2021 STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+properties:
+    drdy-gpios:
+      type: phandle-array
+      required: false
+      description: |
+        DRDY pin
+
+        This pin defaults to active high when produced by the sensor.
+        The property value should ensure the flags properly describe
+        the signal that is presented to the driver.
+
+    odr:
+      type: int
+      required: false
+      default: 0
+      description: |
+          Specify the default output data rate expressed in samples per second (Hz).
+          Default is power-down mode
```

Comment:
Too late to the party to -1, but ...

@avisconti this default is documented, but the reason for it is not justified. This is against policy; can you please document the reason why this is a valid default according to https://docs.zephyrproject.org/latest/guides/dts/bindings.html#general-rules? If this is not the power-on-reset value, this may require changes to comply with policy around defaults.

Reply:
> Too late to the party to -1, but ...
> 
> @avisconti this default is documented, but the reason for it is not justified. This is against policy; can you please document the reason why this is a valid default according to https://docs.zephyrproject.org/latest/guides/dts/bindings.html#general-rules? If this is not the power-on-reset value, this may require changes to comply with policy around defaults.

It is the power-down value. I may fix this description today with a commit. I also have another PR currently under review and I may have done similar mistake, but fortunately I'm still in time for that PR. Thanks for noting it!

Reply:
see #41260